### PR TITLE
Add MetaboLights templates for NMR analyses

### PR DESCRIPTION
**Pull request by StellaEggels**
#29

### CODE REVIEW

**Comment by StellaEggels** [comment]
#91

File: templates/community/TRR341_Phenotyping/Sampling_protocol_for_Assay_file_MIAPPE.xlsx
Comment (Hannah-Doerpholz) [inline templates/community/TRR341_Phenotyping/Sampling_protocol_for_Assay_file_MIAPPE.xlsx:1]: [Sampling_protocol_for_Assay_file_MIAPPE.xlsx](https://github.com/nfdi4plants/Swate-templates/files/15149650/Sampling_protocol_for_Assay_file_MIAPPE.xlsx)

Here too I added some Tag accession number info and REFs

File: templates/community/TRR341_Phenotyping/Growth_protocol_for_Study_file_MIAPPE.xlsx
Comment (Hannah-Doerpholz) [inline templates/community/TRR341_Phenotyping/Growth_protocol_for_Study_file_MIAPPE.xlsx:1]: [Growth_protocol_for_Study_file_MIAPPE.xlsx](https://github.com/nfdi4plants/Swate-templates/files/15149664/Growth_protocol_for_Study_file_MIAPPE.xlsx)

Here too I added some Tag accession number info and REFs

File: templates/community/TRR341_Phenotyping/Aerial_conditions_protocol_for_Study_file_MIAPPE.xlsx
Comment (Hannah-Doerpholz) [inline templates/community/TRR341_Phenotyping/Aerial_conditions_protocol_for_Study_file_MIAPPE.xlsx:1]: [Aerial_conditions_protocol_for_Study_file_MIAPPE.xlsx](https://github.com/nfdi4plants/Swate-templates/files/15149693/Aerial_conditions_protocol_for_Study_file_MIAPPE.xlsx)
Here too I added some Tag accession number info and REFs

**Pull request merged**